front/learn-next-js/src/app/components/CreateSchedules.tsx

```diff
@@ -0,0 +1,10 @@
+async function createSchedules() {
+    const response = await fetch("http://localhost:8000/create-schedules");
```

Comment:
use environment variables for the URL. else deployment in cloud will be hell. read on `NEXT_PUBLIC` and if you need that